src/test/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImplTest.java

```diff
@@ -0,0 +1,188 @@
+package edu.cornell.kfs.pdp.batch.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.apache.commons.io.FileUtils;
+import org.apache.log4j.Logger;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.core.api.util.type.KualiInteger;
+import org.kuali.kfs.pdp.businessobject.AchAccountNumber;
+import org.kuali.kfs.pdp.businessobject.PaymentDetail;
+import org.kuali.kfs.pdp.businessobject.PaymentGroup;
+import org.kuali.kfs.pdp.businessobject.PaymentStatus;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.businessobject.Bank;
+
+import com.rsmart.kuali.kfs.pdp.service.AchBundlerHelperService;
+
+import java.io.BufferedOutputStream;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Calendar;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+
+public class CuExtractPaymentServiceImplTest {
+    
+    private static final String ACH_BATCH_TEST_FILE_PATH = "src/test/java/edu/cornell/kfs/pdp/batch/service";
```

Comment:
I would set this to simply "test" so that you're not creating and removing directories under a path that is under version control.